src/corner-kick/src/components/modules/Visualizer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Subscribe } from 'unstated';
+
+import { TopicListener, TopicListenerType } from '~/components/containers/TopicListener';
+
+import { Canvas } from '~/components/visualizer/Canvas';
+import { Circle } from '~/components/visualizer/Circle';
+import { Rectangle } from '~/components/visualizer/Rectangle';
+
+import { ITurtlePose } from '~/types/standardTopics';
+
+/**
+ * Visualizer module. Display a ball using information from the /ball/pose ROS topic.
+ */
```

Comment:
We should maybe make this comment a bit more generic (we could display many things, balls, field, robots, etc.)